avm/res/compute/virtual-machine/main.bicep

```diff
@@ -617,10 +617,10 @@ resource vm_autoShutdownConfiguration 'Microsoft.DevTestLab/schedules@2018-09-15
             notificationLocale: contains(autoShutdownConfig, 'notificationLocale')
               ? autoShutdownConfig.notificationLocale
               : 'en'
-            timeInMinutes: contains(autoShutdownConfig, 'notificationWebhookUrl')
+            webhookUrl: contains(autoShutdownConfig, 'notificationWebhookUrl')
               ? autoShutdownConfig.notificationWebhookUrl
               : ''
-            webhookUrl: contains(autoShutdownConfig, 'notificationTimeInMinutes')
+            timeInMinutes: contains(autoShutdownConfig, 'notificationTimeInMinutes')
```

Comment:
Same suggestion for all autoshutdown properties. It's preferable to keep them aligned to the original name

Reply:
@eriqua it is a sub property of notifications, thus the parent property before